modules/nomad-cluster/main.tf

```diff
@@ -28,11 +28,19 @@ resource "aws_autoscaling_group" "autoscaling_group" {
   health_check_grace_period = "${var.health_check_grace_period}"
   wait_for_capacity_timeout = "${var.wait_for_capacity_timeout}"
 
-  tag {
-    key                 = "${var.cluster_tag_key}"
-    value               = "${var.cluster_name}"
-    propagate_at_launch = true
-  }
+  tags = [
+    {
+      key                 = "Name"
+      value               = "${var.cluster_name}"
+      propagate_at_launch = true
+    },
```

Comment:
I'm guessing this added `Name` tag is a copy/paste error from the other PR? The `cluster_tag_key` is set to `Name` by default.

Reply:
Yep, shameless copypasta is even more shameless :fearful: 

I'll commit a fix and address #9 at the same time.